frameit/lib/frameit/editor.rb

```diff
@@ -59,6 +62,10 @@ def rotation_for_device_orientation
       return 0
     end
 
+    def should_skip?
+      return should_add_title? && !fetch_text(:title)
```

Comment:
Why do we skip screenshots that do not have a title? 
Shouldn't the condition be "does not have a filter defined"?

Reply:
This is to preserve the existing behavior of "Easy Mode".  See https://github.com/fastlane/fastlane/blob/22f845f82317933635044d55d13d146c8afae431/frameit/lib/frameit/editor.rb#L40

Reply:
This way we only skip if we are in 'complex' mode *and* there is no `filter` defined for the screenshot in the Framefile.json

Reply:
Could/should we rename `should_add_title?` to `is_complex_framing_mode?` or something similar maybe? Or do we loose the "what should happen then" context then?

> there is no `filter` defined for the screenshot in the Framefile.json

This bit I don't understand yet. Is this expressed via `!fetch_text(:title)` in the code? Isn't `title` one level deeper than `filter`?

Reply:
That's a great suggestion, I'll rename `should_add_title?` to `is_complex_framing_mode?` 👍 

Regarding the check for `filter`, you're right in that it is being done by the check on `!fetch_text(:title)`.  What's happening is the conditional that was deleted on line 487 (previously that was throwing a `user_error`) is now the conditional here to see if it should skip.